addons/main/script_mod.hpp

```diff
@@ -1,5 +1,5 @@
 #define MAINPREFIX z
-#define PREFIX tfar_autoswitch
+#define PREFIX grad_tfarAutoswitch
```

Comment:
```suggestion
#define PREFIX grad_tfarautoswitch
```

but I'd go with lowercase to be consisent with `hemtt.toml` and also … maybe filesystem capitalization issues?

Reply:
I used camelCase everywhere, like it is for all our other projects. The only exception from this rule is the `prefix` field in the `hemtt.toml`, to make sure the PBOs are all lowercase to prevent filesystem capitalization issues.